soc/nxp/imx/imx9/imx95/pinctrl_soc.h

```diff
@@ -27,23 +27,23 @@ extern "C" {
 #define IOMUXC_CFGREG   (IOMUXC_BASE + 0x204)
 #define IOMUXC_DAISYREG (IOMUXC_BASE + 0x408)
 
-#define IOMUXC_INPUT_SCHMITT_ENABLE_SHIFT IOMUXC_SW_PAD_CTL_PAD_HYS_SHIFT
-#define IOMUXC_DRIVE_OPEN_DRAIN_SHIFT     IOMUXC_SW_PAD_CTL_PAD_OD_SHIFT
-#define IOMUXC_BIAS_PULL_DOWN_SHIFT       IOMUXC_SW_PAD_CTL_PAD_PD_SHIFT
-#define IOMUXC_BIAS_PULL_UP_SHIFT         IOMUXC_SW_PAD_CTL_PAD_PU_SHIFT
-#define IOMUXC_SLEW_RATE_SHIFT            IOMUXC_SW_PAD_CTL_PAD_FSEL1_SHIFT
-#define IOMUXC_DRIVE_STRENGTH_SHIFT       IOMUXC_SW_PAD_CTL_PAD_DSE_SHIFT
+#define MCUX_IMX_INPUT_SCHMITT_ENABLE_SHIFT IOMUXC_SW_PAD_CTL_PAD_HYS_SHIFT
+#define MCUX_IMX_DRIVE_OPEN_DRAIN_SHIFT     IOMUXC_SW_PAD_CTL_PAD_OD_SHIFT
```

Comment:
I don't understand why we are changing the prefix name here?  The initial macro names are just fine.

Reply:
This is because the imx-rgpio driver expects these prefixes see
https://github.com/zephyrproject-rtos/zephyr/blob/895d91169e8b6e573058a8a3f9fdf081f1362c81/drivers/gpio/gpio_mcux_rgpio.c#L119
Also this would be identical to imx93, imx8m etc pinctrl definitions.